Web/starter_project_iconics/components/layout/Header.tsx

```diff
@@ -0,0 +1,118 @@
+import Image from 'next/image'
+import { useState } from 'react'
+import { MdOutlineClose, MdViewHeadline } from 'react-icons/md'
+import logo from '../../public/a2sv-logo.png'
+
+function classNames(...classes: string[]) {
+  return classes.filter(Boolean).join(' ')
+}
+
+const Header = () => {
+  const [navigation, setNavigation] = useState(() => [
+    { id: 0, name: 'Home', to: '/', current: true },
+    { id: 1, name: 'Teams', to: '/team/teams', current: false },
+    {
+      id: 2,
+      name: 'Success Stories',
+      to: '/Success_story/success_stories',
+      current: false,
+    },
+    { id: 3, name: 'About Us', to: '/about/about', current: false },
+    { id: 4, name: 'Blogs', to: '/blog/blogs', current: false },
+    {
+      id: 5,
+      name: 'Get Involved',
+      to: '/get_involved/get_involved',
+      current: false,
+    },
+  ])
+
+  const [showNav, setShowNav] = useState(false)
+
+  const handleNavClick = (e: React.FormEvent) => {
+    let id = parseInt((e.target as HTMLAnchorElement).id)
```

Comment:
[nit] Idk where id is being modified, so better to use const

Reply:
i think we don't need this too if we remove the id
you can just have them link to their pages and set the `current` based on the current route